site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -327,6 +327,11 @@ The extension is now fully functional! Reload the extension, refresh this page,
 click the button to turn it green! However, some users may want to change the background to a
 different color.
 
+{% Aside 'gotchas' %}
+You can't script any internal Chrome page, like "chrome://extensions". Be sure to try out the
```

Comment:
Ditto.

Reply:
Changed to "You can't run a content script on an internal Chrome page, such as ...".